### ISSUE TITLE
TypeError: Unable to write page when calling writePage

### ISSUE THREAD

**Issue by muhammadhananraza** [galkahana/HummusJS#484]
TypeError: Unable to write page when calling pdfWriter.writePage(current_page);

The error occurs occasionally (even with same data it works fine)

My initial hunch was resource (memory) limitation but that is not the case.

What could be the possible reason(s)?

**Comment by galkahana** [comment]
disk is not available? file cannot be written to?

**Comment by galkahana** [comment]
i dont know. looks ok.

**Comment by muhammadhananraza** [comment]
@galkahana does the library use/access disk internally?

**Comment by galkahana** [comment]
No. Itll write to a file only if you asked it to, and read from it only if you gave it something to read (images, fonts, etc.)
